### PR TITLE
updating yaml name in file

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket

https://jira.d2iq.com/browse/COPS-7244

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4524.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md [hunk 1]
```diff
@@ -45,18 +45,17 @@ kubectl label secret $CLUSTER_NAME-ssh-key clusterctl.cluster.x-k8s.io/move=
 
 ## Create overrides
 
-If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you wish to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
+If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you want to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
 
 ```yaml
```
Comment (branden) [inline pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:50]:
```suggestion
```bash
```

The below changes turn this text snippet into a shell command.

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md [hunk 2]
```diff
@@ -60,17 +60,17 @@ secret/preprovisioned-example-ssh-key labeled
 
 ## Create overrides
 
-If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you wish to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
+If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you want to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
 
 ```yaml
```
Comment (branden) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:65]:
```suggestion
```bash
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md [hunk 3]
```diff
@@ -45,18 +45,17 @@ kubectl label secret $CLUSTER_NAME-ssh-key clusterctl.cluster.x-k8s.io/move=
...
 image_registries_with_auth:
 - host: "registry-1.docker.io"
   username: "my-user"
   password: "my-password"
   auth: ""
-  identityToken: ""
-
-epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
-
+  identityToken: ""epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
```
Comment (branden) [inline pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:57]:
```suggestion
  identityToken: ""
  epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md [hunk 4]
```diff
@@ -60,17 +60,17 @@ secret/preprovisioned-example-ssh-key labeled
...
+cat > overrides.yaml << EOF 
 image_registries_with_auth:
 - host: "registry-1.docker.io"
   username: "my-user"
   password: "my-password"
   auth: ""
-  identityToken: ""
-
-epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
+  identityToken: ""epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
```
Comment (branden) [inline pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:72]:
```suggestion
  identityToken: ""
  epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md [hunk 5]
```diff
@@ -60,17 +60,17 @@ secret/preprovisioned-example-ssh-key labeled
...
+cat > overrides.yaml << EOF 
 image_registries_with_auth:
 - host: "registry-1.docker.io"
   username: "my-user"
   password: "my-password"
   auth: ""
-  identityToken: ""
-
-epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
+  identityToken: ""epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
```
Comment (branden) [inline pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:72]:
```suggestion
  identityToken: ""
  epel_centos_7_rpm: https://my-rpm-repostory.org/epel/epel-release-latest-7.noarch.rpm
```

File: pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md [hunk 6]
```diff
@@ -60,17 +60,17 @@ secret/preprovisioned-example-ssh-key labeled
 
 ## Create overrides
 
-If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you wish to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
+If your pre-provisioned machines have [overrides](../../../image-builder/override-files) you must create a secret that includes all of the overrides you wish to provide in one file. For example, if you want to provide an override with Docker credentials and a different source for EPEL on a CentOS7 machine, you can create a file like this:
 
 ```yaml
```
Comment (branden) [inline pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:65]:
```suggestion
```bash
```

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:50]: true true  - but - if you keep the yaml code bracket, the colors look cool in the UI ;)

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md (hunk 2 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:65]: same here - 
true true - but - if you keep the yaml code bracket, the colors look cool in the UI ;)

File: pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md (hunk 6 above)
Comment (msdolbey) [reply pages/dkp/konvoy/2.3/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:65]: same here
true true - but - if you keep the yaml code bracket, the colors look cool in the UI ;)

**Comment by msdolbey** [comment]
@branden - Good catch - I'll add those changes to the yaml now.

**Review by msdolbey — APPROVED** [review]
👍 
mirror work

File: pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.1/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:50]: Yeah I think we've talked about this before and we like the coloring, right? Still copies well?

File: pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-secrets-and-overrides/index.md:65]: I like colors myself. :)

**Comment by msdolbey** [comment]
@cmastr - yep - still copies! :)

**Pull request merged**